app/src/main/java/com/nutomic/syncthingandroid/model/FolderStatus.java

```diff
@@ -27,4 +27,12 @@ public class FolderStatus {
     public long version;
     public String error;
     public String watchError;
+
+    public boolean isNeedsItems() {
+        return needFiles + needDirectories + needSymlinks + needDeletes > 0;
+    }
+
+    public boolean isOutOfSync() {
+        return state.equals("idle") && isNeedsItems();
+    }
```

Comment:
Given you aren't using `isNeedsItems` anymore, I'd prefer if you removed that and folded it into `isOutOfSync`.